extensions/npm/src/npmMain.ts

```diff
@@ -8,7 +8,7 @@ import * as vscode from 'vscode';
 import { addJSONProviders } from './features/jsonContributions';
 import { runSelectedScript, selectAndRunScriptFromFolder } from './commands';
 import { NpmScriptsTreeDataProvider } from './npmView';
-import { invalidateTasksCache, NpmTaskProvider, hasPackageJson } from './tasks';
```

Comment:
I think this is the only use of the `hasPackageJson` utility function and it could be deleted now.

Reply:
👍